lib/components/Checkbox/Checkbox.test.tsx

```diff
@@ -195,3 +196,25 @@ describe('Checkbox', () => {
     expect(checkbox.checked).toBe(false);
   });
 });
+
+describe('resolveCheckedGroup', () => {
```

Comment:
Feel like we should be testing via the public API rather than an internal util. What do you think?

Reply:
Yeah i think you're right. Legacy of previous iterations 😄